lib/extensions.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const path = require('path');
+
+module.exports = {
+
+  withConflictExclusions: function(BaseGenerator) {
+    return class extends BaseGenerator {
+      constructor(args, opts) {
+        super(args, opts);
+
+        this._conflictExclusions = [];
+        this._extendConflicter();
+      }
+
+      _updateFileContent(filePath, pipeline, options) {
+        const { force } = options || {};
+
+        super._updateFileContent(filePath, pipeline);
+
+        if (force === true) {
+          this._conflictExclusions.push(filePath);
+        }
+      }
+
+      _extendConflicter() {
```

Comment:
That looks like direct patching, let's try to make it more presentable, smth like

```

const Conflicter = require('yeoman-generator/lib/util/conflicter');

class EnhancedConflicter extends Conflicter {

    constructor(args, opts) {
        super(args, opts);
        
        this.forceMap = {};
    }

    collision(file, cb) {
        if (!forceMap.contains(file)) {
            super.collision(file, cb);
            return;
        }

        this.adapter.log.force(path.relative(process.cwd(), file.path));
        cb('force');
    }
}

module.exports =  EnhancedConflicter;

```

Reply:
And then set enhanced conflicter in consturctor

Reply:
The original `Conflicter` class is not exported from module - I didn't find better solution rather that use this way.